guide/token-formats.md

```diff
@@ -0,0 +1,99 @@
+
+We use this format for authorization codes and refresh tokens. They are only meant to be consumed by OpenIddict itself.
+
+ ### Use cases
+
+  - When there's no separate resource server
+  - When resource servers are .NET only and can share ASP.NET Core's DataProtection configuration with auth server
+
+  ### Benefits
+  - No additional configuration required
+  - Uses OpenIddict's built-in validation
+  - Resource servers can validate tokens without having to contact auth server
+  - Tokens are encrypted so no one can inspect the token, e.g. if tokens somehow end up in your logs somewhere or are intercepted somehow
+
+  ### Drawbacks
+  - Proprietary format, so if you add non .NET core resource servers in the future you need to swith to JWTs
+  - Claims are stored within the token, which is convenient but token size could get large if there are a lot of claims (probably not an issue in real-world scenerios)
+  - Token expiration is in the token itself, so even if users sign out their tokens will still be valid until they reach their expiration
+
+  ### Setup and API validation configuration
+   (link to configuration)
+
+
+## Reference tokens
+   
+   When using reference token format, authorization codes, access tokens and refresh tokens are stored as ciphertext in the database and a crypto-secure random identifier is returned to the client application. 
+   
+   ### Use cases 
+   
+    - When resource servers are .NET only and can access OpenIddict's data store
+    - If you need to keep track of all the issued tokens
+    - If you need to store a very large number of claims in the authorization codes, access tokens and refresh tokens
+    - When immediate revocation of reference access tokens is desired
+    
+
+  ### Benefits
+   
+   - Minimal configuration required
+   - Uses OpenIddict's built-in validation
+   - Resource servers can validate tokens without having to contact auth server
+   - Token sizes are very small because they only contain ids
+   - See use cases above
+
+  ### Drawbacks
+   - Requires a connection to OpenIddict's data store, e.g. Entity Framework DataContext. Resource servers may not want to have to reference OpenIddict's database
+   - Because only ids are in the access tokens, a call to the database is required for every request
+
+  ### Setup and API validation configuration
+   (link to configuration)
+
+## JWTs (JSON Web Tokens)
+
+ These are standard tokens verifiable by third parties, used by Azure Active Directory, Google, Facebook, Auth0, etc and any valid OAuth 2.0 service. They are signed by the authorization server but their contents are not encrypted so they can be read by anyone.
+
+ ### Use cases
+
+  - When you need interopability with non-.NET core apis / resource servers, e.g. Node, Java, PHP, etc
+
+### Benefits
+
+  - Good to be familiar with JWTs because they are the standard token type in OAuth 2.0
```

Comment:
The OAuth2 specification says the access token format is not defined - and up to the implementation to decide what it should be - so we can't say JWT is the standard one, even if it's statically a very frequent choice.